stl/inc/chrono

```diff
@@ -5451,6 +5507,28 @@ namespace chrono {
         }
         return _Os << static_cast<unsigned int>(_Val) << _STATICALLY_WIDEN(_CharT, " is not a valid month");
     }
+
+    template <class _CharT, class _Traits>
+    basic_ostream<_CharT, _Traits>& operator<<(basic_ostream<_CharT, _Traits>& _Os, const year& _Val) {

```

Comment:
Should we try to reuse `_Try_write` for the month overload above?

Reply:
The default month format is `%b`, which is an abbreviated spelling of the month (e.g. Jan), so it has to go through `put_time`. The invalid month handling also doesn't align with any formatters (months are 2 digits, but the spec just wants a static cast printed, so month `0` would be `"0"` and not `"00"`).